client/my-sites/checkout/controller.jsx

```diff
@@ -149,17 +149,15 @@ export function conciergeQuickstartSession( context, next ) {
 	);
 
 	const state = context.store.getState();
-	const selectedSite = getSelectedSite( state );
-
-	if ( ! selectedSite ) {
-		return null;
-	}
+	const selectedSiteId = getSelectedSiteId( state );
 
 	context.primary = (
 		<CartData>
 			<ConciergeQuickstartSession
-				receiptId={ Number( receiptId ) }
-				selectedSiteId={ selectedSite.ID }
+				receiptId={ receiptId }
+				selectedSiteId={ selectedSiteId }
+				path={ context.path }
+				reduxStore={ context.store }
```

Comment:
The `reduxStore` prop is no longer needed after the component stopped using `redirectIfLoggedOut`.

Reply:
Thanks, taken care of in [9849222](https://github.com/Automattic/wp-calypso/pull/33749/commits/98492222ecd61d3624a0558dcc8394047785e873).